cmds/core/ip/neigh_linux.go

```diff
@@ -6,33 +6,34 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
 	"math"
 	"net"
+	"sort"
+	"strconv"
 	"strings"
 
 	"github.com/vishvananda/netlink"
 )
 
-const neighHelp = `Usage: ip neigh { add | del | replace }
-                ADDR [ lladdr LLADDR ] [ nud STATE ] 
-                [ dev DEV ] [ router ] [ extern_learn ] 
+const neighHelp = `Usage: 	ip neigh { add | del | replace }
```

Comment:
```suggestion
const neighHelp = `Usage: 	ip neigh { add | delete | replace }
```

Reply:
I've kept it the same as the original command which lists it as 'del' but also accepts delete.

Reply:
I see, thanks for clarifying.